From b088294b4c7e4bf433ec9a655a0d873c674576b5 Mon Sep 17 00:00:00 2001
From: Aleksandr Korzun <a.korzun@partner.samsung.com>
Date: Fri, 23 Jun 2023 14:58:23 +0300
Subject: [PATCH 33/33] [git] Add .gitignore

Add a .gitignore file that has been missing since splitting off DeepRecSys
from PNMLibrary.

Signed-off-by: Aleksandr Korzun <a.korzun@partner.samsung.com>
---
 .gitignore | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..b354ac9
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,3 @@
+__pycache__
+*.pyc
+*.log
-- 
2.34.1

